scipy/stats/_distn_infrastructure.py

```diff
@@ -1054,7 +1054,7 @@ def rvs(self, *args, **kwds):
             self._random_state = random_state_saved
 
         # Cast to int if discrete
-        if discrete and not isinstance(self, rv_sample):
+        if discrete and not isinstance(self, _rv_sample):
```

Comment:
I suppose we could leave the name of `rv_sample` alone now that the public class is `rv_count`. Not necessary to change back; just a thought.

Reply:
Good points, `rv_sample` is back in the last commit.